### ISSUE TITLE
Surrogation output

### ISSUE THREAD

**Issue by knightperfectionist** [nedap/deidentify#56]
I am getting following issue while surrogating a document

```
2021-03-05 12:41:41.242 | WARNING  | deidentify.surrogates.dataset_deidentifier:generate_surrogates:87 - Cannot apply corpus shuffle for tag=DATE:DATE as there are no choices. Ensure that there are at least two distinct DATE:DATE annotations inseparate documents. Will now fall-back to identity replacement.
2021-03-05 12:41:41.244 | WARNING  | deidentify.surrogates.dataset_deidentifier:generate_surrogates:87 - Cannot apply corpus shuffle for tag=NAME:PATIENT as there are no choices. Ensure that there are at least two distinct NAME:PATIENT annotations inseparate documents. Will now fall-back to identity replacement.
2021-03-05 12:41:41.245 | WARNING  | deidentify.surrogates.dataset_deidentifier:generate_surrogates:87 - Cannot apply corpus shuffle for tag=ID:MEDICALRECORD as there are no choices. Ensure that there are at least two distinct ID:MEDICALRECORD annotations inseparate documents. Will now fall-back to identity replacement.
2021-03-05 12:41:41.246 | WARNING  | deidentify.surrogates.dataset_deidentifier:generate_surrogates:87 - Cannot apply corpus shuffle for tag=AGE:AGE as there are no choices. Ensure that there are at least two distinct AGE:AGE annotations inseparate documents. Will now fall-back to identity replacement.
2021-03-05 12:41:41.247 | WARNING  | deidentify.surrogates.dataset_deidentifier:generate_surrogates:87 - Cannot apply corpus shuffle for tag=LOCATION:HOSPITAL as there are no choices. Ensure that there are at least two distinct LOCATION:HOSPITAL annotations inseparate documents. Will now fall-back to identity replacement.
2021-03-05 12:41:41.249 | WARNING  | deidentify.surrogates.dataset_deidentifier:generate_surrogates:87 - Cannot apply corpus shuffle for tag=PROFESSION:PROFESSION as there are no choices. Ensure that there are at least two distinct PROFESSION:PROFESSION annotations inseparate documents. Will now fall-back to identity replacement.
2021-03-05 12:41:41.251 | WARNING  | deidentify.surrogates.dataset_deidentifier:generate_surrogates:87 - Cannot apply corpus shuffle for tag=NAME:DOCTOR as there are no choices. Ensure that there are at least two distinct NAME:DOCTOR annotations inseparate documents. Will now fall-back to identity replacement.
2021-03-05 12:41:41.252 | WARNING  | deidentify.surrogates.dataset_deidentifier:generate_surrogates:87 - Cannot apply corpus shuffle for tag=CONTACT:PHONE as there are no choices. Ensure that there are at least two distinct CONTACT:PHONE annotations inseparate documents. Will now fall-back to identity replacement.
  
```

@jantrienes

**Comment by jantrienes** [comment]
Dear Pushkar, 

surrogates are generated as per the following strategy:

1. Get a tag-specific surrogate generator (if present) https://github.com/nedap/deidentify/blob/dc1119b2322626448b1cea9e41a705231d616380/deidentify/surrogates/generators/__init__.py#L25-L41
2. If no tag-specific generator has been defined, shuffle the annotations across the corpus (`shuffle_generator`) https://github.com/nedap/deidentify/blob/dc1119b2322626448b1cea9e41a705231d616380/deidentify/surrogates/generators/__init__.py#L46-L47
3. If a PHI is unique in the corpus, shuffling cannot take place. So it falls-back to replacing a PHI with itself (no-op). 

I noticed that your tagging schema (e.g., `DATE:DATE`, `NAME:PATIENT`) is different from the ones we used. So you probably want to replace the mapping in `GeneratorFactory`

Does this help?
